gulpfile.js

```diff
@@ -37,7 +38,10 @@ var tsProject =
 gulp.task('compile', function() {
   hasError = false;
   var tsResult =
-      gulp.src(['lib/**/*.ts', 'typings/**/*.d.ts', 'node_modules/typescript/lib/typescript.d.ts'])
+      gulp.src([
+            'lib/**/*.ts', 'typings/**/*.d.ts', 'node_modules/typescript/lib/typescript.d.ts',
+            'node_modules/dart-style/dart-style.d.ts'
```

Comment:
this should not be necessary, TypeScript should find this just from the `require('dart-style');`.